azext_iot/tests/test_iot_ext_unit.py

```diff
@@ -664,20 +663,9 @@ def test_device_module_list(self, serviceclient, top):
         args = serviceclient.call_args
         url = args[0][0].url
         method = args[0][0].method
-        body = args[0][2]
-        headers = args[0][1]
-
-        assert method == 'POST'
-        assert '{}/devices/query?'.format(mock_target['entity']) in url
-        assert body['query'] == "select * from devices.modules where devices.deviceId = '{}'".format(device_id)
-        assert json.dumps(result)
+        assert method == 'GET'
+        assert '{}/devices/mydevice/modules?'.format(mock_target['entity']) in url
```

Comment:
mydevice should be `'{}'.format(device_id)`

Reply:
fixed